site/layouts/partials/examples/main.html

```diff
@@ -19,16 +19,16 @@ <h2 id="{{ $entry.category | urlize }}">{{ $entry.category }}</h2>
           <div>
             <h3 class="h5 mb-1">
               <a class="d-block link-offset-1" href="{{ urls.JoinPath $.Site.Params.github_org $example.url }}" target="_blank" rel="noopener">
-                {{ $example.name }}
+                <span id="starter{{ $i }}">{{ $example.name }}</span>
```

Comment:
Just a detail, but for consistency with IDs defined in examples in the `/docs/5.3/components/`, I'd use the `edit-{{ $i }} starter-{{ $i }}` format instead of `edit{{ $i }} starter{{ $i }}` to generate `edit-0 starter-0` instead of `edit0 starter0`